ETC Course: 11. ETC Social and Physical Security (#1095)
ethereumclassic/ethereumclassic.github.io: content/blog/2023-03-02-ethereum-classic-course-11-ethereum-classic-social-and-physical-layer-security/index.md

In this class we will explain what are the physical and social layers, their security models in more detail, how they work combined, and what are the possible attack types in each layer.

## 2. The Physical Layer vs the Social Layer 

![The Physical Layer and Social Layer.](./1.png)

The physical layer, which is the live operating network, is the most well known and consists of the protocol rules, the software clients that people must run in their machines to participate, cryptography, and the energy PoW uses that protects the blockchain.

The social layer consists of all the node operators (e.g. exchanges, wallets, dapps, startups, Node as a Service (NaaS) providers, block explorers, and independent participants), mining pools, and independent miners. It also includes the developers who conduct research and development to advance the protocol and fix bugs when necessary. All these constituents, plus users and investors who own the native cryptocurrency, decide whether to adopt changes through an improvement proposal process and by running the protocol through software clients.

> Although the physical layer may be highly secure and indestructible, the truth is that if all the constituents in the network at the social layer were to agree on bad changes or corrupt practices, they could, technically, do it. 

Reviewer: The problem found in this sentence is the lack of an appropriate object 
 referenced by the pronoun "it" in the last phrase. There is not an adequate action in the sentence which answers the question "do what"? As written, the only action to which the pronoun can refer is "agree". By removing all prepositional phrases and replacing pronouns with the object referenced, the redundancy and loss of meaning becomes easier to see.
"... truth is ... constituents ... agree"
The action referenced by "do it" is not present in the sentence. To leave the action implied is not recommended.
```suggestion
Although the physical layer may be highly secure and indestructible, the truth is that if all the constituents in the network at the social layer were to agree on bad changes or corrupt practices, they could, technically, implement them.
```

> However, this is extremely difficult because blockchain communities are usually very conservative and principled ecosystems and there is usually ample activism that analyzes and filters proposals and potential changes. This is why the combined systems of the social and physical layers are the complete story of how Ethereum Classic is protected.

Reviewer: Line 37: Usage
```suggestion
However, this is extremely difficult because blockchain communities are usually very conservative and principled ecosystems where there is usually ample activism that analyzes and filters proposals and potential changes. This is why the combined systems of the social and physical layers are the complete story of how Ethereum Classic is protected.
```

But, how do the security models in each layer work? And, how do they work combined?

## 3. Physical Layer: Proof of Work Security

![The physical layer.](./2.png)

As we mentioned before, proof of work is the key security component of the physical layer, or live operating network.

> The way it works is that there must be demand for ETC’s decentralized computing. If there is demand, then people will buy the cryptocurrency to use it, and as an investment, making the price rise. When the ETC price rises, then mining becomes more profitable. When miners becomes more profitable, they put more machines with more hashing power to build blocks for the network. When more hashing power is dedicated to the network, then it becomes more secure. And, when ETC is more secure, then demand for its decentralized computing grows, which, in turn, makes the price go up further, continuing the virtuous circle.

Reviewer: Line 47: Rewrite for clarity and agreement
```suggestion
In order for the system to work, there must be demand for ETC’s decentralized computing. When there is demand, people buy the cryptocurrency to use it and as an investment causing the price to rise. When the ETC price rises, mining becomes more profitable. When mining becomes more profitable, miners place more machines in service adding more hashing power to build blocks for the network. With more hashing power dedicated to the ETC network, it becomes more secure. When ETC is more secure, demand for its decentralized computing grows, which, in turn, makes the price go up further, thus continuing the virtuous circle.
```

## 4. Social Layer: Coordination Problem Security

![The social layer.](./3.png)

> On the social layer, the key security component is that for all the constituents who participate in the network (i.e. exchanges, wallets, dapps, startups, Node as a Service (NaaS) providers, block explorers, independent participants, mining pools, and independent miners); who reside in different parts of the world, in different cultures, with different languages, and different worldviews; it is extremely difficult to coordinate to attack the network, censor it, or corrupt it. 

Reviewer: Line 53: Rewrite for clarity
```suggestion
On the social layer, the key security component is that all the constituents who participate in the network (i.e. exchanges, wallets, dapps, startups, Node as a Service (NaaS) providers, block explorers, independent participants, mining pools, and independent miners) reside in different parts of the world, in different cultures, with different languages and worldviews making it is extremely difficult to coordinate an attack on the network, or to censor or corrupt it. 
```

> However, the economic incentives to use the network are conducive of increased use, as demand for decentralized computing grows. If demand for decentralized computing grows, then more dapp developers will build useful things on ETC. As more useful dapps will exist on the blockchain, then more users will be attracted to it. If more users are attracted to it, then more node operators will participate. The more node operators participate, from different parts of the world, the more difficult it is to coordinate attacks, making the blockchain even more secure. And, when ETC is more secure, then demand for its decentralized computing grows, which in turn makes more dapp developers build on ETC, continuing the virtuous circle.

Reviewer: Line 55: Rewrite for clarity
```suggestion
The economic incentives to use the network are conducive to increased use as demand for decentralized computing grows. When demand for decentralized computing grows, more dapp developers build useful things on ETC. The more useful dapps that exist on the blockchain, the more users are attracted to it. As more users are attracted to ETC, then more node operators will participate. The more node operators participating from different parts of the world, the more difficult it becomes to coordinate attacks, making the blockchain even more secure. And when ETC is more secure, the demand for its decentralized computing grows, encouraging even more dapp developers to build on ETC, again continuing this virtuous circle.
```

## 5. The ETC Security Model Combined

![The ETC security model combined.](./4.png)

> The way these two security virtuous circles work combined; with proof of work as the key component at the live operating network, and the coordination problem as the key component at the social layer; is that both have incentives to increase price and participation, and as both sides grow, then demand for ETC computing services grows, which increases price and participation even more.

Reviewer: Line 61: Rewrite for clarity
```suggestion
These two virtuous circles of security work with Nakamoto consensus proof of work as the key component of the live operating network and the difficulty of attack coordination as the key component of the social layer so that both layers give incentive to increasing price and participation. As both sides grow, demand for ETC computing services grows which increases price and participation even more.
```

## 6. Social Layer Attacks and Defense Mechanisms

> But, what are the possible attacks at the social layer and what are the defense mechanisms?

Reviewer: Line 65: Remove unneeded comma
```suggestion
But what are the possible attacks at the social layer and what are the defense mechanisms?
```

> **Proposals for bad changes:** These are fixes or upgrades that reduce decentralization. Usually these proposals need to go through the Ethereum Classic Improvement Proposal process (ECIP). The way the global ecosystem defends ETC against this type of attack is by opposing and not approving such changes, or not adopting them if for some reason they are approved.

Reviewer: Line 67: Correct word order
```suggestion
**Proposals for bad changes:** These are fixes or upgrades that reduce decentralization. Usually these proposals need to go through the Ethereum Classic Improvement Proposal (ECIP) process. The way the global ecosystem defends ETC against this type of attack is by opposing and not approving such changes, or not adopting them if for some reason they are approved.
```

> **Proposals for bad changes:** These are fixes or upgrades that reduce decentralization. Usually these proposals need to go through the Ethereum Classic Improvement Proposal process (ECIP). The way the global ecosystem defends ETC against this type of attack is by opposing and not approving such changes, or not adopting them if for some reason they are approved.

Reviewer: Line 67: Rewrite for clarity
```suggestion
**Proposals for bad changes:** These are fixes or upgrades that reduce decentralization. Usually these proposals need to go through the Ethereum Classic Improvement Proposal (ECIP) process. The way the global ecosystem defends ETC against this type of attack is by opposing and not approving such changes or, if for some reason they are approved at the organization level, not adopting them at the individual participant level.
```

> **Censorship:** When Ethereum migrated to proof of stake, it became more than 60% censored by the US Office of Foreign Assets Control (OFAC) nearly immediately because staking pools are centralized. The way ETC prevents this is by having miners join and leave the network whenever they want, wherever they are in the world, taking advantage of the lowest electricity prices possible wherever they may be. This level of decentralization is only possible with the attributes of proof of work explained in previous classes and this is why the PoW mining business is practically impossible to capture.

Reviewer: The implication of "having miners join and leave" is that miners are being directed to do so by ETC. The proof of work protocol makes this possible but does not include an impetus to the miner to join or leave.

> **External restrictions of use:** As mining is decentralized an practically impossible to capture, then external restrictions such as know your customer (KYC) and anti-money laundering (AML) prohibitions are only effective to users in their own geographies. But, as long as transactions or a smart contracts are sent to ETC from anywhere at anytime, and they comply with protocol rules, ETC will always process and execute them. 

Reviewer: Line 71: Correct spelling; Usage
```suggestion
**External restrictions of use:** As mining is decentralized and practically impossible to capture, then external restrictions such as know your customer (KYC) and anti-money laundering (AML) prohibitions are only effective upon users in their own jurisdictional geographies. So long as transactions or smart contracts are sent to ETC, from anywhere at any time, that comply with protocol rules, ETC will always process and execute them. 
```

**Backward compatibility:** Sometimes there are upgrades that break past smart contracts. These must be stopped at the social layer and are stopped in the same way as proposals for bad changes. 

*Important: In the unlikely scenario that the worldwide social layer entirely agrees to integrate bad changes, censorship, external restrictions, or upgrades that break backward compatibility, then there would be permanent losses of security. The only recourse left by individual users would be to pursue restitution through normal traditional systems such as the legal process and law enforcement, but this would be very difficult to achieve because of the decentralized nature of the system.*

## 7. Physical Layer Attacks and Defense Mechanisms

Because proof of work is such a powerful safety mechanism, the most plausible attacks may involve the most recent transactions rather than medium to long term past activity in the network.

> **51% attacks:** These attacks consist of accumulating 50% or more of the computing power of the mining sector in ETC and then sending malicious blocks where very recent transactions are deleted, thus stealing money from the victims. The best defense against this kind of attack is by increasing the computing power dedicated to the network, but this is difficult to do in the short term. The other way is to wait for more confirmations than usual to make sure that, when ETC are received, it is highly unlikely that the transactions involved  will be reversed.  

Reviewer: Line 81: Spacing
```suggestion
**51% attacks:** These attacks consist of accumulating 50% or more of the computing power of the mining sector in ETC and then sending malicious blocks where very recent transactions are deleted, thus stealing money from the victims. The best defense against this kind of attack is by increasing the computing power dedicated to the network, but this is difficult to do in the short term. The other way is to wait for more confirmations than usual to make sure that, when ETC are received, it is highly unlikely that the transactions involved will be reversed.  
```
